src/main/java/filter/expression/Qualifier.java

```diff
@@ -243,6 +243,8 @@ public boolean isSatisfiedBy(IModel model, TurboIssue issue, MetaQualifierInfo i
             return satisfiesUpdatedHours(issue);
         case REPO:
             return satisfiesRepo(issue);
+        case COUNT:
+                return satisfiesCount();
```

Comment:
`count` is a meta-qualifier (like `in` and `sort`), so it should be [stripped](https://github.com/HubTurbo/HubTurbo/blob/master/src/main/java/filter/expression/Qualifier.java#L405) by the time we get here. This method shouldn't have to be changed.